Engine/platform/windows/acplwin.cpp

```diff
@@ -133,6 +125,14 @@ struct AGSWin32 : AGSPlatformDriver {
   virtual bool LockMouseToWindow();
   virtual void UnlockMouse();
 
+#ifndef AGS_NO_VIDEO_PLAYER
+  virtual void PlayVideo(const char* name, int skip, int flags);
+  virtual void PauseApplication();
```

Comment:
Please don't do this. These functions have very certain meaning, not related to video.

Reply:
Fair enough. I checked before hand and the only places these functions were implemented was for video playback. I figured we could break them out of the #ifdef block if we used them for more than just managing video.